docs/run/retired-systems.mdx

```diff
@@ -71,4 +73,19 @@ jobs = provider.backend.jobs(backend_name=<backend_name>)
 
 #If you want to retrieve a specific job you have the id for
 job = provider.backend.retrieve_job(<job_id>)
+```
+
+### For a job run from Qiskit Runtime
+
+```python
+from qiskit_ibm_runtime import QiskitRuntimeService
+
+# Load your IBM Quantum account(s). Replace "hub/group/project" with your desired instance
+service = QiskitRuntimeService(channel="ibm_quantum", instance="hub/group/project")
+
+# Retrieve a single job by id
+job = service.job("<job_id>")
+
+# Retrieve a batch of jobs. Filtering options can be found in the QiskitRuntimeService.jobs api reference
+jobs = service.jobs(backend_name="<backend>")
```

Comment:
Again, probably just safest to remove the quotations to match the IBMProvider code snippet formatting.